app/src/main/res/layout/item_artist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="80dp"
```

Comment:
it looks like the same layout but difference in dimensions. You can extract dimensions to dimen.xml for portrait and landscape and keep just one layout